src/pybind/mgr/dashboard/frontend/src/app/core/create-cluster-wizard/create-cluster-wizard.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="container h-75">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="create-cluster">
+      <h2 class="create-cluster-title">
+          Please create your first cluster
+      </h2>
+
+      <div class="offset-md-3">
+        <button class="btn btn-accent m-2"
+                [routerLink]="'/dashboard'"
+                i18n>Create Cluster</button>
+        <button class="btn btn-light"
+                [routerLink]="'/dashboard'"
+                i18n>Skip</button>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
What about following [Patternfly's empty state layout](https://www.patternfly.org/v4/components/empty-state/html)?

I'd avoid having styles called 'create-cluster' or equivalent, as that's mixing presentation with contents. For example, Patternfly uses `pf-c-title` or `pf-c-empty-state__body`.

Reply:
Have you considered creating a generic component for wizards?
- Empty state page
- 1 page, 2 page, etc? You may have a look at the Telemetry one for reference.
- Final/wrap-up page.

Reply:
I am exploring its possibility in a separate branch. If you are okay, then I can do it in the next PR with the 1st page.